.github/workflows/helm.yml

```diff
@@ -234,6 +235,7 @@ jobs:
           path: substra-charts
 
       - name: Publish chart
+        if: ${{ ! github.event_name != 'pull_request' }}
```

Comment:
Did you test this condition with an alpha ? not sure taht this will publish it as it will be a workflow dispatch

Reply:
I'm not sure that was meant to stay here actually. Is it ok if I just remove it?

Reply:
yep :)